schema/annotation-source.yaml

```diff
@@ -147,7 +152,17 @@ $defs:
       An InformationEntity representing an individual piece of data, generated/acquired through methods 
       which reliably produce truthful information about something.
     properties:
-      value:
+      type:
```

Comment:
Adding a type field for DataItem (to match type field in all other classes)

Reply:
Good catch!